package.json

```diff
@@ -65,7 +65,7 @@
     "rollup-plugin-uglify": "^6.0.2",
     "semantic-release": "^17.1.1",
     "sinon": "^7.0.0",
-    "uglify-js": "^2.0.0",
+    "uglify-js": "^3.4.9",
```

Comment:
3.4.9 is the version used inside rollup as well?

Reply:
Yes, that's the specified version inside their package but due to caret range (^), once installed both theirs and ours will resolve to exactly `3.10.2`, which is already what we use today. This resolved version should be locked by `yarn.lock`.

`uglify-js` as required by `rollup-plugin-uglify`: https://github.com/amplitude/Amplitude-JavaScript/blob/main/yarn.lock#L6983-L6991

`uglify-js` installed version: https://github.com/amplitude/Amplitude-JavaScript/blob/main/yarn.lock#L8153-L8156

Reply:
Sounds good